tests/src/main/java/org/hibernate/beanvalidation/tck/tests/time/ClockProviderTest.java

```diff
@@ -148,7 +169,20 @@ public void canConfigureClockProviderForValidator() {
 		assertCorrectNumberOfViolations( constraintViolations, 0 );
 	}
 
+	private static void checkClockProviderHasDefaultProperties(ClockProvider clockProvider) {
+		Clock clock = clockProvider.getClock();
+		Clock systemClock = Clock.systemDefaultZone();
+
+		if ( !systemClock.getZone().equals( clock.getZone() ) ) {
+			throw new RuntimeException( "The default clock provider should use the default system time zone." );
```

Comment:
At first, they were included in the ClockProvider implementation and it made more sense to use a RuntimeException but you're right, I changed it.